code/src/Components/Form.js

```diff
@@ -0,0 +1,153 @@
+                        />}
+          {step === 2 && <EmailQuestion 
+                            step = {step}
+                            emailInput={emailInput} 
+                            onEmailInputChange={onEmailInputChange} 
+                            onStepChange={onStepChange}
+                            onStepBackChange={onStepBackChange}
+                        />}
+          {step === 3 && <PositionQuestion 
+                            step = {step}
+                            positionInput={positionInput}
+                            onPositionInputChange={onPositionInputChange} 
+                            onStepChange={onStepChange}
+                            onStepBackChange={onStepBackChange}
+                        />}
+          {step === 4 && <OfficeQuestion 
+                            step = {step}
+                            officeInput={officeInput}
+                            onOfficeInputChange={onOfficeInputChange} 
+                            onStepChange={onStepChange}
+                            onStepBackChange={onStepBackChange}
+                        />}
+          {step === 5 && <SkillsQuestion 
+                            step = {step}
+                            skillsArray={skillsArray}
+                            onSkillsArrayChange={onSkillsArrayChange}
+                            onStepChange={onStepChange}
+                            onStepBackChange={onStepBackChange}
+                        />}
+          {step === 6 && <ExperienceQuestion 
+                            step = {step}
+                            experienceLevel={experienceLevel}
+                            onExperienceLevelChange={onExperienceLevelChange}
+                            experienceTitle = {experienceTitle}
+                            onStepChange={onStepChange}
+                            onStepBackChange={onStepBackChange}
+                        />}
+          {step === 7 && <LinkQuestion 
+                            step = {step}
+                            linkInput={linkInput}
+                            onLinkInputChange={onLinkInputChange}
+                            onStepChange={onStepChange}
+                            onStepBackChange={onStepBackChange}
+                        />}
+          {step === 8 && <Overview 
+                            step = {step}
+                            nameInput={nameInput} 
+                            emailInput={emailInput} 
+                            positionInput={positionInput} 
+                            officeInput={officeInput} 
+                            skillsArray={skillsArray} 
+                            linkInput={linkInput}
+                            experienceTitle={experienceTitle}
+                            onStepBackChange={onStepBackChange}
+                        />}   
+          {step === 9 && <ThanksPage
+                        />}  
+      </form>
+
+
```

Comment:
this is a very big file, you have a lot of questions so it would probably be nice to consider dividing the form as well as the components to make it cleaner/easier to find what you are looking for :)